bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/i18n/fenecon.properties

```diff
@@ -0,0 +1 @@
+# 
```

Comment:
Please create contents for this file with: `mvn i18n:generate-default-translations`